metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -69,6 +70,11 @@ class BuildServerConnection private (
   // hasDebug is not yet available in BSP capabilities
   // https://github.com/build-server-protocol/build-server-protocol/pull/161
   def hasDebug: Boolean = isBloop || isSbt
```

Comment:
I know you didn't touch this _but_ seeing we are talking about relating things. I'm assuming that both of these set that the are `runProvider` and `debugProvider` in the `buildServerCapabilities`. Couldn't we just grab that instead of explicitly checking for `isBloop` or `isSbt`?

Reply:
Oh yes indeed. However sbt does not set the `debugProvider` field. So this should be fixed in sbt beforehand, but the `buildServerCapabilities` in sbt may not be overriden easily...